Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -120,6 +120,27 @@
   #define IS_RRD_SC
   #define IS_U8GLIB_SSD1306
 
+#elif ENABLED(FYSETC_242_OLED_12864)
+      
+  #define IS_RRD_SC
+  #define U8GLIB_SH1106
```

Comment:
@GerogeFu — This disagrees with the block in `marlin_DOGM.h` which tries to use either SSD1306 or SSD1309 for `FYSETC_242_OLED_12864`. It looks like displays with these controllers can have SPI or I2C interfaces, so I don't want to presume they always refer to I2C. As we're trying to wrangle display pins and types, this is one of those head-scratchers.